lib/note-detail.jsx

```diff
@@ -90,7 +91,11 @@ export default React.createClass( {
 
 	renderMarkdown( divStyle ) {
 		const { content = '' } = this.props.note.data;
-		const markdownHTML = marked( content );
+		const markdownHTML = marked( content, {
+			highlight: function( code ) {
+				return highlight.highlightAuto( code ).value;
+			}
+		} );
```

Comment:
might start out by pulling this function into the module scope instead of creating a new closure on each call. could also use fat-arrow syntax for functions.

```js
const highlighter = code => highlight.highlightAuto( code ).value;

…

const markdownHTML = marked( content, { highlight: highlighter } );
```